### PR TITLE
contributing: Recommend pipenv run over python -m

### PR DESCRIPTION
**Pull request by underyx**
# Thanks for improving Semgrep Docs 😀

### Please ensure

- [ ] A subject matter expert (SME) reviews the content
- [ ] A technical writer reviews the content or PR
- [ ] This change has no security implications or else you have pinged the security team
- [ ] Redirects are added if the PR changes page URLs

### CODE REVIEW

File: docs/contributing/semgrep-contributing.md [hunk 1]
```diff
@@ -148,16 +142,10 @@ docker build -t semgrep .
...
+To run tests, run the following command:
 
 ```
-python -m pytest
-```
-
-This command will run comprehensize parse tests on many open source projects. To skip these slow tests run:
-
-```sh
-python -m pytest --ignore=tests/qa/test_public_repos.py
```
Comment (underyx) [inline docs/contributing/semgrep-contributing.md:160]: I'm adding a default ignore for this so won't be needed

**Comment by adamkvitek** [comment]
@underyx thanks for doing this! Btw. can you try help me to run the test suit? I was not successful until now.

File: docs/contributing/semgrep-contributing.md [hunk 2]
```diff
@@ -84,16 +84,10 @@ Alternatively, you may include it somewhere like `/usr/local/bin/`.
 
 ## Running `semgrep`
 
-You will want to be in the pipenv environment whenever you run semgrep. Start a shell with
+Make sure you are in `semgrep/semgrep/`, then run:
```
Comment (brendongo) [inline docs/contributing/semgrep-contributing.md:87]:
```suggestion
Make sure you are in `cli/`, then run:
```

File: docs/contributing/semgrep-contributing.md [hunk 3]
```diff
@@ -84,16 +84,10 @@ Alternatively, you may include it somewhere like `/usr/local/bin/`.
 
 ## Running `semgrep`
 
-You will want to be in the pipenv environment whenever you run semgrep. Start a shell with
+Make sure you are in `cli/`, then run:
```
Comment (adamkvitek) [inline docs/contributing/semgrep-contributing.md:87]:
```suggestion
Ensure that you are in `cli/` directory, then issue the following command:
```

**Comment by adamkvitek** [comment]
@brendongo @underyx hello, any update on this PR?

**Pull request merged**